src/connector.js

```diff
@@ -868,7 +879,7 @@ function updateResultListState( newState ) {
 				.replace( '%[short-date-fr]', getShortDateFormat( resultDate ) )
 				.replace( '%[long-date-en]', getLongDateFormat( resultDate, 'en' ) )
 				.replace( '%[long-date-fr]', getLongDateFormat( resultDate, 'fr' ) )
-				.replace( '%[highlightedExcerpt]', highlightedExcerpt );
+				.replace( '%[highlightedExcerpt]', highlightedExcerpt.replace( '&amp;' , '&' ).replace( '&' , '&amp;' ) );
```

Comment:
@bblaisATcoveo This will not work as expected in scenarios where there would be other special characters such as "é" (&eacute;). Has this been taken into consideration?

Reply:
@GormFrank Right, it's not the excerpt that we want to fix but just the printable Uri and make sure we encode the & to &amp;, code adjusted